projects/1.20-fabric/assets/species/species/lang/zh_cn.json

```
@@ -0,0 +1,245 @@
+  "item.species.cracked_wraptor_egg": "破碎的诡异速龙蛋",
+  "item.species.wraptor_spawn_egg": "诡异速龙刷怪蛋",
+
+  "effect.species.wither_resistance": "凋零抗性",
+
+  "subtitles.entity.species.wraptor.aggro": "诡异速龙：叫声",
+  "subtitles.entity.species.wraptor.agitated": "诡异速龙：咯咯叫",
+  "subtitles.entity.species.wraptor.attack": "诡异速龙：啄击",
+  "subtitles.entity.species.wraptor.death": "诡异速龙：死亡",
+  "subtitles.entity.species.wraptor.hurt": "诡异速龙：受伤",
+  "subtitles.entity.species.wraptor.idle": "诡异速龙：咯咯叫",
+  "subtitles.entity.species.wraptor.shear": "剪羽毛声",
+  "subtitles.entity.species.wraptor.step": "诡异速龙：闲逛",
+  "subtitles.entity.species.wraptor.feather_loss": "诡异速龙：失去羽毛",
+  "subtitles.entity.species.wraptor.egg": "诡异速龙：下蛋",
+  "subtitles.block.species.wraptor_egg.crack": "诡异速龙蛋：裂开",
+  "subtitles.block.species.wraptor_egg.hatch": "诡异速龙蛋：孵化",
+  "subtitles.item.species.cracked_wraptor_egg.slurp": "啜食声",
+
+  "advancements.species.shear_wraptor_completely.title": "聪明姑娘",
+  "advancements.species.shear_wraptor_completely.description": "给一只诡异速龙剪光羽毛并面对后果",
+  "advancements.species.hatch_wraptor.title": "沉重的爱",
+  "advancements.species.hatch_wraptor.description": "用铁砧压碎一个诡异速龙蛋",
+
+  "entity.species.deepfish": "深潜鱼",
+  "item.species.deepfish_spawn_egg": "深潜鱼刷怪蛋",
+  "item.species.deepfish_bucket": "深潜鱼桶",
+
+  "subtitles.entity.species.deepfish.flop": "深潜鱼：翻滚",
+  "subtitles.entity.species.deepfish.death": "深潜鱼：死亡",
+  "subtitles.entity.species.deepfish.hurt": "深潜鱼：受伤",
+  "subtitles.entity.species.deepfish.idle": "深潜鱼：嚎叫",
+
+  "advancements.species.deepfish_in_a_bucket.title": "深海之歌",
+  "advancements.species.deepfish_in_a_bucket.description": "用桶抓住一只深潜鱼",
+
+  "entity.species.roombug": "屋虫",
+  "item.species.roombug_spawn_egg": "屋虫刷怪蛋",
+
+  "subtitles.entity.species.roombug.eat": "屋虫：进食",
+  "subtitles.entity.species.roombug.death": "屋虫：死亡",
+  "subtitles.entity.species.roombug.hurt": "屋虫：受伤",
+  "subtitles.entity.species.roombug.idle": "屋虫：咔哒声",
+  "subtitles.entity.species.roombug.snoring": "屋虫：打呼噜",
+  "subtitles.entity.species.roombug.step": "屋虫：踏步",
+
+  "entity.species.birt": "鵭",
+  "entity.species.birt_egg": "鵭蛋",
+  "item.species.birt_spawn_egg": "鵭刷怪蛋",
+  "item.species.birt_egg": "鵭蛋",
+  "block.species.birt_dwelling": "鵭巢",
+
+  "item.species.music_disc_dial": "音乐唱片",
+  "item.species.music_disc_dial.desc": "bunten - dial",
+
+  "effect.species.birtd": "鵭之眩晕",
+
+  "subtitles.entity.species.birt.death": "鵭：死亡",
+  "subtitles.entity.species.birt.hurt": "鵭：受伤",
+  "subtitles.entity.species.birt.idle": "鵭：啁啾声",
+  "subtitles.entity.species.birt.fly": "鵭：飞行",
+  "subtitles.entity.species.birt.message": "鵭：交流",
+  "subtitles.effect.birtd.applied": "鵭：引起混乱",
+  "subtitles.block.species.birt_dwelling.collect": "鵭：收集蛋",
+  "subtitles.block.species.birt_dwelling.enter": "鵭：进入巢穴",
+  "subtitles.block.species.birt_dwelling.exit": "鵭：离开巢穴",
+  "subtitles.block.species.birt_dwelling.work": "鵭：密谋中",
+  "subtitles.item.species.birt_egg.throw": "扔出鵭蛋",
```

```suggestion
  "subtitles.item.species.birt_egg.throw": "鵭蛋：投掷",
```
有动作主体或客体的字幕建议还是按这个格式
全文通查